ImageProcessorMaven/src/main/java/de/telran/service/CreateOriginName.java

```diff
@@ -0,0 +1,11 @@
+package de.telran.service;
+
+public class CreateOriginName {
+    public static String createOriginNameFromUrl(String urlName){
```

Comment:
It is used only once in the main method. You may think of a NamingService, which will encapsulate the logic of names generation for images.